webpack/services/mock.js

```diff
@@ -0,0 +1,218 @@
+// eslint-disable-next-line import/prefer-default-export
+export const translate = a => a;
```

Comment:
Why did we mock this out?  I believe that using `__()` should work in katello already because it is defined in foreman.

Reply:
Yeah I wasn't quite sure on this. We'll want to mock it at least for the tests I suppose but I'll change it back to the double underscore.